stories/dooboo-ui/Accordion.stories.tsx

```diff
@@ -12,81 +12,240 @@ const Container = styled.View`
   justify-content: center;
 `;
 
-const StlyedImage = styled.Image`
-  height: 25px;
-  width: 25px;
-  margin-right: 8px;
+const StyledImage = styled.Image`
+  width: 20px;
+  height: 20px;
+`;
+
+const StyledTitle = styled.Text`
+  font-weight: bold;
+  color: #ffffff;
+  padding-left: 20px;
+  padding-right: 20px;
+`;
+
+const StyledItem = styled.Text`
+  font-weight: bold;
+  color: black;
+`;
+
+const CustomStyledTitle = styled.Text`
+  font-weight: bold;
+  color: #ffffff;
+  position: absolute;
+  left: 50px;
+`;
+
+const CustomStyledItem = styled.Text`
+  font-weight: bold;
+  color: black;
+  position: absolute;
+  left: 50px;
+`;
+
+const LeftElement = styled.Image`
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  left: 20px;
+`;
+
+const RightElement = styled.Image`
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  right: 20px;
 `;
 
 const Default = (): React.ReactElement => {
-  const accordionData = [
+  const data = [
     {
-      itemTitle: '커스텀 패널',
-      itemBodies: ['패널 커스텀', '패널 커스텀', '패널 커스텀'],
+      title: {
+        name: <StyledTitle>Defualt-title-01</StyledTitle>,
+      },
+      bodies: [
+        {
+          name: <StyledItem>Default body01</StyledItem>,
+        },
+        {
+          name: <StyledItem>Default body02</StyledItem>,
+        },
+      ],
     },
     {
-      itemTitle: '판매 관리',
-      itemBodies: ['판매처 관리', '공급처 관리', '상품 관리', '재고 관리'],
+      title: {
+        name: <StyledTitle>Defualt-title-02</StyledTitle>,
+      },
+      bodies: [
+        {
+          name: <StyledItem>Default body01</StyledItem>,
+        },
+        {
+          name: <StyledItem>Default body02</StyledItem>,
+        },
+      ],
     },
     {
-      itemTitle: '주문 배송 관리',
-      itemBodies: ['주문 보기', '주문 보기', '주문 보기', '주문 보기'],
+      title: {
+        name: <StyledTitle>Defualt-title-03</StyledTitle>,
+      },
+      bodies: [
+        {
+          name: <StyledItem>Default body01</StyledItem>,
+        },
+        {
+          name: <StyledItem>Default body02</StyledItem>,
+        },
+      ],
     },
   ];
 
   return (
     <Container>
       <Accordion
-        data={accordionData}
+        data={data}
         isAnimated={true}
-        collapseOnStart={false}
-        animDuration={300}
+        collapseOnStart={true}
+        animDuration={400}
         activeOpacity={1}
+        toggleElement={<StyledImage source={ARROW_UP} />}
```

Comment:
Could you use `svg` you've created here instead of the images? I think you can delete this images then.

Reply:
@DevYuns

Reply:
Please open up another `PR` for this.